contracts/controller/PowerEnabled.sol

```diff
@@ -91,9 +82,7 @@ contract PowerEnabled is MarketEnabled {
     uint256 amountPow = _amountBabz.mul(authorizedPow).div(totalBabz);
     // check pow limits
     uint256 outstandingPow = outstandingPower();
-    require(outstandingPow.add(amountPow) <= maxPower);
```

Comment:
@bkcninja aren't we still need this check?

Reply:
True @troggy.